x-pack/solutions/security/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/task_runner.ts

```diff
@@ -254,7 +255,7 @@ export default function createBackfillTaskRunnerTests({ getService }: FtrProvide
 
       // query for alert docs
       const alertDocs = await queryForAlertDocs<SecurityAlert>(es, alertsAsDataIndex);
-      expect(alertDocs.length).to.eql(9);
+      expect(alertDocs.length).to.eql(10);
```

Comment:
This test started to fail after relocation with actual `alertDocs.length` being `10`. I would like to get test code owner confirmation what can cause it and if the change is acceptable.